Integreat/Integreat.Localization/AppResources.fa.resx

```diff
@@ -259,4 +259,10 @@
   <data name="WhereAreYou" xml:space="preserve">
     <value>تو کجایی؟</value>
   </data>
+  <data name="RefreshOptions" xml:space="preserve">
+    <value>#TODO#_زمانی که محتوای ممکن است به روز شود انتخاب کنید</value>
```

Comment:
das sollten wir bis geklärt ist ob es passt noch durch die englische Version ersetzen oder leer lassen

Reply:
I think we should leave it empty

Reply:
Yes than the English text will be used